### PR TITLE
feat:#269 Add faq guide after first login 

### PR DESCRIPTION
**Pull request by cryindevil**

### CODE REVIEW

File: lib/widget/dialogs/login_dialog.dart [hunk 1]
```diff
@@ -112,6 +113,7 @@ class LoginDialogState extends State<LoginDialog> {
           widget.personInfo.value = newInfo;
           progressDialog.dismiss(showAnim: false);
           Navigator.of(context).pop();
+          showFAQ();
```
Comment (w568w) [inline lib/widget/dialogs/login_dialog.dart:116]: Well, you are showing this after ***every***, not ***first*** login... Perhaps add a preference item to record this?

File: lib/widget/dialogs/login_dialog.dart [hunk 2]
```diff
@@ -140,6 +142,32 @@ class LoginDialogState extends State<LoginDialog> {
     }
   }
 
+  Future<bool?> showFAQ() {
+    return showDialog<bool>(
```
Comment (w568w) [inline lib/widget/dialogs/login_dialog.dart:146]: Change to `showPlatformDialog`. 

[We never use `showDialog`](https://github.com/search?q=repo%3ADanXi-Dev%2FDanXi%20showDialog&type=code), since it always shows a Material-style dialog on all platforms.

**Review by w568w — CHANGES_REQUESTED** [review]

File: lib/widget/dialogs/login_dialog.dart (hunk 1 above)
Comment (cryindevil) [reply lib/widget/dialogs/login_dialog.dart:116]: Unable to track first time of login in. Log out from the UIS account will wipe out local preferences.

File: lib/widget/dialogs/login_dialog.dart (hunk 2 above)
Comment (cryindevil) [reply lib/widget/dialogs/login_dialog.dart]: fixed

Comment (w568w) [reply lib/widget/dialogs/login_dialog.dart]: > fixed

You needn't reply indeed. Use the `Resolve conversation` button below.

And when you feel the PR suitable for re-review, click `Re-request review` button beside the reviewer's avatar on the right column.

**Comment by w568w** [comment]
And... `fix: ` prefix is usually abused for every single thing fixed. It should not be. 

Please take a look at an ad-hoc commit message guide here: https://github.com/DanXi-Dev/DanXi/commit/3262b64b61b8670c8879663df73ea1a140804ff5.

File: lib/l10n/intl_en.arb [hunk 3]
```diff
@@ -530,5 +530,10 @@
   "manually_add_course_message": "If there is a class that is not included in the class schedule, but you want to check it, add it manually yourself.",
   "start_date_select": "Semester start date query",
   "start_date_select_message": "You must select the start date of the semester to view your class schedule. Not selecting will result in a date error. \nTip: Start date is the Monday of the first week.",
-  "empty_classroom_warning": "Not connected to the campus intranet, so classroom usage information may not be accurate.\nPlease connect to the campus intranet for accurate information."
+  "empty_classroom_warning": "Not connected to the campus intranet, so classroom usage information may not be accurate.\nPlease connect to the campus intranet for accurate information.",
+  "welcome_prompt":"If you are using DanXi for the first time, perhaps you would like to take a look at the FAQ section? \n\nWelcome to the DanXi!",
+  "have_read":"have read",
+  "read":"read",
```
Comment (w568w) [inline lib/l10n/intl_en.arb:536]: The `"read"` is used nowhere. Consider removing it.

File: lib/l10n/intl_en.arb [hunk 4]
```diff
@@ -530,5 +530,10 @@
...
+  "faq_url":"https://danxi-dev.feishu.cn/wiki/wikcnrPPGDCiTODBYRkdwLlHH65"
```
Comment (w568w) [inline lib/l10n/intl_en.arb:537]: It'd better put URLs in `constant.dart`, not in translation strings. E.g.:

https://github.com/DanXi-Dev/DanXi/blob/3262b64b61b8670c8879663df73ea1a140804ff5/lib/common/constant.dart#L67-L72

File: lib/l10n/intl_en.arb [hunk 5]
```diff
@@ -530,5 +530,10 @@
   "manually_add_course_message": "If there is a class that is not included in the class schedule, but you want to check it, add it manually yourself.",
   "start_date_select": "Semester start date query",
   "start_date_select_message": "You must select the start date of the semester to view your class schedule. Not selecting will result in a date error. \nTip: Start date is the Monday of the first week.",
-  "empty_classroom_warning": "Not connected to the campus intranet, so classroom usage information may not be accurate.\nPlease connect to the campus intranet for accurate information."
+  "empty_classroom_warning": "Not connected to the campus intranet, so classroom usage information may not be accurate.\nPlease connect to the campus intranet for accurate information.",
+  "welcome_prompt":"If you are using DanXi for the first time, perhaps you would like to take a look at the FAQ section? \n\nWelcome to the DanXi!",
+  "have_read":"have read",
```
Comment (w568w) [inline lib/l10n/intl_en.arb:535]: "Have read" does not seem an appropriate word here. Maybe reuse `S.of(cxt).skip` or rephrase it to "I have read"?

File: lib/l10n/intl_en.arb [hunk 6]
```diff
@@ -530,5 +530,10 @@
   "manually_add_course_message": "If there is a class that is not included in the class schedule, but you want to check it, add it manually yourself.",
   "start_date_select": "Semester start date query",
   "start_date_select_message": "You must select the start date of the semester to view your class schedule. Not selecting will result in a date error. \nTip: Start date is the Monday of the first week.",
-  "empty_classroom_warning": "Not connected to the campus intranet, so classroom usage information may not be accurate.\nPlease connect to the campus intranet for accurate information."
+  "empty_classroom_warning": "Not connected to the campus intranet, so classroom usage information may not be accurate.\nPlease connect to the campus intranet for accurate information.",
+  "welcome_prompt":"If you are using DanXi for the first time, perhaps you would like to take a look at the FAQ section? \n\nWelcome to the DanXi!",
```
Comment (w568w) [inline lib/l10n/intl_en.arb:534]: I suggest swapping two paragraphs, i.e.

```
Welcome to the DanXi!

If you are using DanXi for the first time, perhaps you would like to take a look at the FAQ section? 
```

**Review by w568w — CHANGES_REQUESTED** [review]

**Review by w568w — APPROVED** [review]
